lib/src/test/java/io/ably/lib/test/other/OtherSuite.java

```diff
@@ -0,0 +1,21 @@
+package io.ably.lib.test.other;
+
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.RunWith;
+import org.junit.runner.notification.Failure;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        HttpUtilsTest.class,
+})
+public class OtherSuite {
```

Comment:
I think it would be ok for these tests to be part of the rest suite, since they're tests for the http primitive. If it is a separate suite, the lets call it `UtilsSuite` and not `OtherSuite`.